owncloudApp/src/main/res/values/strings.xml

```diff
@@ -755,4 +756,5 @@
     <string name="ocis_accounts_warning_checkbox_message">Don\'t show again</string>
     <string name="ocis_accounts_warning_button">Understood</string>
 
+    <string name="file_already_exists_description">File with name %1$s, already exists.</string>
```

Comment:
```suggestion
    <string name="file_already_exists_description">File with name %1$s already exists.</string>
```
I think that comma doesn't make sense here, and web client doesn't use it either